db/1.10/modules/ROOT/pages/database-connector-driver-configuration.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Driver Visibility using Mule Maven Plugin
+
+Anypoint Studio helps you configure database driver dependencies without problems, though there are some scenarios where you must use the Mule Maven Plugin to set up properly the driver classes visibility because of class loading isolation mechanisms.
```

Comment:
```suggestion
Although Anypoint Studio enables you to configure database driver dependencies without problems in most cases, there are some scenarios where you must use the Mule Maven Plugin to properly set up the driver classes' visibility because of class loading isolation mechanisms.
```